src/Entity/Server.php

```diff
@@ -59,7 +62,10 @@
  *     "endpoint",
  *     "debug_flag",
  *     "caching",
- *     "batching"
+ *     "batching",
+ *     "disable_introspection",
+ *     "query_depth",
+ *     "query_complexity"
```

Comment:
Thanks, but we should have all config properties on the class as properties. for consistency, all other properties are there as well.